src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -214,8 +214,6 @@ internal void Filter()
                 }
                 strBuilder.Append(", ");
             }
-
-            Analytics.LogPiiInfo("Filter-categories", strBuilder.ToString().Trim());
```

Comment:
When is this triggered?

Reply:
Analytics.LogPiiInfo is deprecated and actually does not log anything anymore. I just cleaned up the SearchViewModel class ...I do not think performance is affected by this though..